app/controllers/sessions_controller.rb

```diff
@@ -1,5 +1,6 @@
 class SessionsController < ApplicationController
   def show
+    @user = current_user
```

Comment:
You can use current_user in the show page without assigning an instance variable.